### DISCUSSION TITLE
The selection of dataTable doesnt work well with virtualscroll

### DISCUSSION THREAD

**Discussion by kaisor6688** [primefaces/community#1359] [PrimeVue]
In a datatable i can use keyboard arrowDown/arrowUp to move the mark and use Enter to select the row.
If the datatable has virtualscroll on and after clicking arrowDown serveral times and the internal lazyloading feature is triggered, the mark will not be contiunous increased, but jump far away below.

for example. I mark the first row of a datatable and keep clicking the arrow down to 21, 22, 23...30, 31  Somehow the lazyloading is triggered, and now if click the arrow down again, it will jump to 50 instead of 32.

(tested in latest chrome/edge)

any advice?

**Comment by tugcekucukoglu** [comment]
Can not test it on showcase. Do you have a reproducer link?
